nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/AbstractPutHBase.java

```diff
@@ -170,7 +193,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
     }
 
     protected String getTransitUri(PutFlowFile putFlowFile) {
-        return "hbase://" + putFlowFile.getTableName() + "/" + putFlowFile.getRow();
+        return "hbase://" + putFlowFile.getTableName() + "/" + new String(putFlowFile.getRow());
```

Comment:
we may want to do `new String(putFlowFile.getRow(), StandardCharsets.UTF8)` just to be safe